src/views/v2/default-template.php

```diff
@@ -15,5 +15,25 @@
 use Tribe\Events\Views\V2\Template_Bootstrap;
 
 get_header();
+
+/**
+ * Provides an action that allows for the injection of HTML at the top of the template after the header.
+ *
+ * @since TBD
+ *
+ * @param string $eventDisplay The string representation (slug) of the displayed view - "month".
+ */
+do_action( 'tribe_default_events_views_v2_template_after_header', $eventDisplay );
```

Comment:
Event Display is not a variable in V2. And on this case we can use a entry point here.

Reply:
Changed the param, but entry_point isn't available here.